bitnami/redis/templates/redis-node-statefulset.yaml

```diff
@@ -370,8 +374,19 @@ spec:
           resources: {{- toYaml .Values.metrics.resources | nindent 12 }}
   {{- end }}
       {{- $needsVolumePermissions := and .Values.volumePermissions.enabled .Values.slave.persistence.enabled .Values.securityContext.enabled .Values.containerSecurityContext.enabled }}
-      {{- if or $needsVolumePermissions .Values.sysctlImage.enabled }}
+      {{- if or $needsVolumePermissions (or .Values.sysctlImage.enabled .Values.sentinel.service.createMaster) }}
       initContainers:
+        {{- if .Values.sentinel.service.createMaster }}
+        - name: kubectl
+          image: bitnami/kubectl
+          imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
+          command: ["cp", "/opt/bitnami/kubectl/bin/kubectl", "/kubectl"]
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: kubectl
+              mountPath: /kubectl
+        {{- end }}
```

Comment:
In this case, the initcontainer should be fully templatized (check below other examples). This means the image parameters should be added to the values, create the proper template in the helpers, etc